### PR TITLE
update tidb roadmap

### PR DESCRIPTION
**Pull request by Oreoxmt**
### What is changed, added or deleted? (Required)

Update TiDB roadmap
### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [ ] v7.5 (TiDB 7.5 versions)
- [ ] v7.4 (TiDB 7.4 versions)
- [ ] v7.3 (TiDB 7.3 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs/pull/15194
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: tidb-roadmap.md
Comment (Oreoxmt) [inline tidb-roadmap.md:1]: Preview: https://scaffold-git-preview-pingcapdocs-cn15441-oreoxmt.vercel.app/zh/tidb/dev/tidb-roadmap

File: tidb-roadmap.md [hunk 1]
```diff
@@ -69,61 +70,26 @@ TiDB 路线图展示了 TiDB 未来的计划。随着我们发布长期稳定版
...
-      <td>
-        <ul>
-          <li>
-            <b>兼容 MySQL 8.0</b>
-          </li>
-          <br />
           <li>
-            <b>为数据导入、备份恢复、PITR 提供统一的 SQL 接口</b>
+            <b>管控 Runaway Query</b><br />
+              一种由运维人员控制的方式，用于显著提升在出现意外高成本查询的工作负载的性能稳定性
```
Comment (ran-huang) [inline tidb-roadmap.md:75]:
```suggestion
              一种由运维人员控制的方式，显著提升了出现非预期的高成本查询时的性能稳定性
```

File: tidb-roadmap.md [hunk 2]
```diff
@@ -137,42 +103,41 @@ TiDB 路线图展示了 TiDB 未来的计划。随着我们发布长期稳定版
...
-            <br />大幅提高 TiDB 到 TiDB 的数据吞吐量
+            <b>TiCDC 与数据仓库或数据湖系统的集成</b>
+            <br />
           </li>
           <br />
           <li>
-            <b>升级期间自动暂停/恢复 DDL</b>
-            <br />提供平滑的升级体验
+            <b>TiDB 节点标签</b>
+            <br />为 DDL 操作分配现有的或新添加的 TiDB 节点，以将 DDL 任务与在线流量使用的计算资源隔离
```
Comment (ran-huang) [inline tidb-roadmap.md:112]:
```suggestion
            <br />将 DDL 操作分配到现有的或新添加的 TiDB 节点，以便将 DDL 任务与在线流量使用的计算资源隔离
```

File: tidb-roadmap.md [hunk 3]
```diff
@@ -69,61 +70,26 @@ TiDB 路线图展示了 TiDB 未来的计划。随着我们发布长期稳定版
...
-        <b>SQL 功能</b>
-        <br />增强 SQL 功能和兼容性
-      </td>
-      <td>
-        <ul>
-          <li>
-            <b>兼容 MySQL 8.0</b>
-          </li>
-          <br />
           <li>
```
Comment (ran-huang) [inline tidb-roadmap.md:73]:
```suggestion
          <br />
          <li>
```

File: tidb-roadmap.md [hunk 4]
```diff
@@ -210,6 +171,10 @@ TiDB 路线图展示了 TiDB 未来的计划。随着我们发布长期稳定版
             <b>统一的 TLS CA/密钥轮换策略</b>
             <br />统一管理所有 TiDB 组件的证书
           </li>
+          <li>
+            <b>支持 AWS FIPS</b>
+            <br />实现 FedRAMP 合规性
```
Comment (ran-huang) [inline tidb-roadmap.md:176]:
```suggestion
            <br />实现 FedRAMP 合规
```

File: tidb-roadmap.md [hunk 5]
```diff
@@ -226,9 +191,6 @@ TiDB 路线图展示了 TiDB 未来的计划。随着我们发布长期稳定版
             <b>增强数据脱敏</b>
           </li>
           <br />
```
Comment (ran-huang) [inline tidb-roadmap.md:193]:
```suggestion
```

File: tidb-roadmap.md [hunk 6]
```diff
@@ -28,25 +28,22 @@ TiDB 路线图展示了 TiDB 未来的计划。随着我们发布长期稳定版
       <td>
         <ul>
           <li>
-            <b>Partitioned Raft KV 存储引擎 GA</b><br />支持 PB 级别的集群，提升写入速度、扩缩容操作速度，提升数据整理的稳定性
+             <b>分布式并行执行框架</b><br />
+            TiDB v7.2.0 引入了用于后台任务（如 DDL 和 analyze）的分布式并行执行框架。这是实现这些任务在计算节点间并行化的基础。v7.4.0 为分布式 reorg 任务（如 DDL 和 import）引入了全局排序，这大大减少了存储中额外资源的消耗。用户可以使用外部存储简化操作并节省成本
```
Comment (ran-huang) [inline tidb-roadmap.md:32]:
```suggestion
            TiDB v7.2.0 引入了用于后台任务（如 DDL 和 analyze）的分布式并行执行框架，为实现这些任务在计算节点间并行化提供了基础。v7.4.0 为分布式 reorg 任务（如 DDL 和 import）引入了全局排序，大幅减少了存储中额外资源的消耗。用户可以使用外部存储简化操作并节省成本
```

**Review by ran-huang — APPROVED** [review]

**Pull request merged**